### PR TITLE
Update inference-with-keras.ipynb segmentation mask function

### PR DESCRIPTION
**Pull request by divyashreepathihalli**
Improve segmentation mask rendering

### CODE REVIEW

File: site/en/gemma/docs/paligemma/inference-with-keras.ipynb [hunk 1]
```diff
@@ -1,43 +1,16 @@
 {
   "cells": [
-    {
```
Comment (elisabandy) [inline site/en/gemma/docs/paligemma/inference-with-keras.ipynb:3]: the block containing the copyright looks like it's getting lost in the change and should be preserved

File: site/en/gemma/docs/paligemma/inference-with-keras.ipynb [hunk 2]
```diff
@@ -48,7 +21,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
         "<td>\n",
-        "<a target=\"_blank\" href=\"https://ai.google.dev/gemma/docs/paligemma/inference-with-keras\"><img src=\"https://ai.google.dev/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on ai.google.dev</a>\n",
+        "<a target=\"_blank\" href=\"https://ai.google.dev/gemma/docs/paligemma/fine-tuning-paligemma\"><img src=\"https://ai.google.dev/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on ai.google.dev</a>\n",
```
Comment (elisabandy) [inline site/en/gemma/docs/paligemma/inference-with-keras.ipynb:24]: this link should not be changed

File: site/en/gemma/docs/paligemma/inference-with-keras.ipynb [hunk 3]
```diff
@@ -1,43 +1,16 @@
...
-      ]
-    },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "etcMXWCUJApZ"
       },
       "source": [
-        "# Inference with Keras\n"
+        "Project: /gemma/_project.yaml\n",
```
Comment (elisabandy) [inline site/en/gemma/docs/paligemma/inference-with-keras.ipynb:9]: these should not be in the github copy of the notebook

**Review by elisabandy — CHANGES_REQUESTED** [review]

**Comment by joefernandez** [comment]
Closing this PR due to lack of activity. This change is out of sync: recommend restarting update with synced notebook.
